packages/main/test/specs/MultiComboBox.spec.js

```diff
@@ -264,6 +266,82 @@ describe("MultiComboBox general interaction", () => {
 			assert.strictEqual((await list.getProperty("items")).length, 4, "4 items should be shown");
 		});
 
+		it("tests if tokenizer is scrolled to the end when expanded and to start when narrowed", async () => {
+			await browser.url(`test/pages/MultiComboBox.html`);
+
+			const mcb = await $("#more-mcb");
+			const input = mcb.shadow$("input");
+
+			await mcb.scrollIntoView();
+			await input.click();
+
+			let tokenizerScrollContainerScrollLeft = await browser.execute(() => document.querySelector("#more-mcb").shadowRoot.querySelector("ui5-tokenizer").shadowRoot.querySelector(".ui5-tokenizer--content").scrollLeft);
+			let tokenizerScrollContainerScrollWidth = await browser.execute(() => document.querySelector("#more-mcb").shadowRoot.querySelector("ui5-tokenizer").shadowRoot.querySelector(".ui5-tokenizer--content").scrollWidth);
+			let tokenizerScrollContainerClientWidth = await browser.execute(() => document.querySelector("#more-mcb").shadowRoot.querySelector("ui5-tokenizer").shadowRoot.querySelector(".ui5-tokenizer--content").clientWidth);
+			
+			// On the remote test execution there is 1px difference in the scroll position
+			// so the test is adjusted to pass on both remote and local environments
+			let isScrolledToStart = (tokenizerScrollContainerScrollLeft === tokenizerScrollContainerScrollWidth - tokenizerScrollContainerClientWidth) || (tokenizerScrollContainerScrollLeft === tokenizerScrollContainerScrollWidth - tokenizerScrollContainerClientWidth - 1);
```

Comment:
:( why.... does the core team know about that? I don't know if this workaround in the tests is a good idea.

Reply:
I will discuss that with them

Reply:
this is my last comment to be cleared. Everything else looks okay :)